homeassistant/components/conversation/default_agent.py

```diff
@@ -142,6 +150,18 @@ def async_hass_started(hass: core.HomeAssistant) -> None:
 
     start.async_at_started(hass, async_hass_started)
 
+    async def timer_finished_listener(event: core.Event) -> None:
+        """Forward command from timer to process service."""
+        timer_data = event.data.get("data", {})
+        if command := timer_data.get(ASSIST_COMMAND):
+            await hass.services.async_call(
+                DOMAIN,
+                SERVICE_PROCESS,
+                {ATTR_TEXT: command, ATTR_LANGUAGE: event.data[ATTR_LANGUAGE]},
+            )
+
+    hass.bus.async_listen(EVENT_INTENT_TIMER_FINISHED, timer_finished_listener)
```

Comment:
This feels a bit weird. Why is this happening here? Should this be handled by the intent instead?

Reply:
I considered that originally, but it creates a circular dependency between `conversation` and `intent`.